README.md

```
@@ -54,3 +54,8 @@ crontab -e
 ```bash
 python manage.py collectstatic
 ```
+
+# tailwind 명령어
```

markdownlint-cli2 (0.21.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)